packages/scripts/scripts/lint-js.js

```diff
@@ -18,7 +18,7 @@ const {
 
 const args = getCliArgs();
 
-const defaultFilesArgs = ! hasFileInCliArgs ? [ '.' ] : [];
```

Comment:
Couple asides:

- I changed the logic of this expression since if I personally find that "if not x, else" is generally more confusing to follow in than "if x, else". Seems like a good general rule of thumb, but open to counter-points, since maybe it could be argued that as assigning a default value, the most relevant logic path is the absence of the CLI argument. Admittedly I probably shouldn't have made the refactor here vs. getting the fix in place.
- This function is technically not named correctly in accordance with naming guidelines. It should be `hasFileInCLIArgs` (["CLI" is an acronym](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#abbreviations-and-acronyms)).

Reply:
> * This function is technically not named correctly in accordance with naming guidelines. It should be `hasFileInCLIArgs` (["CLI" is an acronym](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#abbreviations-and-acronyms)).

I will fix it in a follow-up PR, I bet there are more functions to update 😅 

I'm fine with "if x, else" it reads better in general.  PR was small enough to include it and [The Boy Scout Rule](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html) always applies 👍

Reply:
> I will fix it in a follow-up PR, I bet there are more functions to update 😅

Ready: #16091.